### PR TITLE
Av 1752 add new platform option

### PR DESCRIPTION
**Pull request by beazil**
- Remove showcase_type from showcase schema
- Remove cli-command create_showcase_type_vocabulary
- Remove references to cli command

### CODE REVIEW

**Review by bzar — APPROVED** [review]

**Comment by beazil** [comment]
I won't merge this pull request yet, as when it gets deployed cli command `ckan -c $APP_DIR/production.ini sixodp-showcase migrate_website_showcase_type_to_platform` should be ran in container
Also reference on `ckan --config /srv/app/production.ini sixodp-showcase create_showcase_type_vocabulary` should be removed from cypress tests when this is merged and showcase-tests needs fixes too

**Comment by beazil** [comment]
New pull request in main repository https://github.com/vrk-kpa/opendata/pull/1580

**Pull request closed without merging**
